Tests/CommandsTests/RunToolTests.swift

```diff
@@ -19,6 +19,18 @@ final class RunToolTests: XCTestCase {
     private func execute(_ args: [String], packagePath: AbsolutePath? = nil) throws -> String {
         return try SwiftPMProduct.SwiftRun.execute(args, packagePath: packagePath, printIfError: true)
     }
+    private func executeTrash(_ args: [String], packagePath: AbsolutePath? = nil) throws {
```

Comment:
Can you think of some other method name?

Reply:
Haha! I had hoped you'd come back with a particular suggestion for a better name :) Ok, I'll try.